src/coreclr/jit/gentree.cpp

```diff
@@ -29442,6 +29442,57 @@ bool GenTree::IsInvariant() const
     return OperIsConst() || OperIs(GT_LCL_ADDR) || OperIs(GT_FTN_ADDR);
 }
 
+//-------------------------------------------------------------------
+// IsVectorPerElementMask: returns true if this node is a vector constant per-element mask
+// (every element has either all bits set or none of them).
+//
+// Returns:
+//     True if this node is a vector constant per-element mask
+//
+bool GenTree::IsVectorPerElementMask(var_types simdBaseType, unsigned simdSize) const
```

Comment:
Perhaps, we can name it as ElementsAreAllBitsSetOrZero as well?

Reply:
We've explicitly used the term `PerElementMask` elsewhere throughout the JIT to encompass the general concept necessary here.